cmd/cosign/cli/verify.go

```diff
@@ -145,7 +149,26 @@ func (c *VerifyCommand) Exec(ctx context.Context, args []string) (err error) {
 	}
 	co.SigVerifier = pubKey
 
-	for _, imageRef := range args {
+	allImgs := []string{}
+	allImgs = append(allImgs, args...)
+
+	for _, f := range c.Files.slice {
+		err := isExtensionAllowed(f)
```

Comment:
Will we distinguish the given file whether `Dockerfile` in this function? Currently, `isExtensionAllowed` only returns _true_ in case given file is `.yml` or `.yaml`. In the flag description, since we pointed out that `-f` flag accepts _Dockerfile_, we actually do not check here.

```golang
flagset.Var(&cmd.Files, "f", "files to validate (kubernetes manifests or Dockerfiles)")
```

Will we move the `verify-dockerfile` into `verify` command in the long term?